tests/FlatBuffers.Test/FlatBuffersExampleTests.cs

```diff
@@ -102,6 +102,17 @@ public void CanCreateNewFlatBufferFromScratch()
             // the mana field should retain its default value
             Assert.AreEqual(monster.MutateMana((short)10), false);
             Assert.AreEqual(monster.Mana, (short)150);
+            
+            // Create an array of sorted tables, can be used with binary search
+            Offset<Monster>[] off = { mon, mon2 };
+            var monsters = Monster.CreateMyTableVector(off, fbb.DataBuffer);
```

Comment:
Why is the sorting done at reading time? If you look at the C++ implementation, it saves the tables in sorted order into the buffer. Then when reading, no sorting is required and the binary search can be done in-place.